types/receipt.go

```diff
@@ -31,12 +31,17 @@ type Receipt struct {
 	GasUsed         uint64
 	ContractAddress Address
 	TxHash          Hash
+	TransactionType TransactionType
```

Comment:
Did we need to put this in the receipt? Why?

Reply:
@zivkovicmilos Please check [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)